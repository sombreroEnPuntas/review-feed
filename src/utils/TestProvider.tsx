import React from 'react'
import { Provider } from 'react-redux'
import { ThemeWrapper } from 'retro-ui'

import store from '../data'

const TestProvider = ({ children }) => (
  <ThemeWrapper>
    <Provider store={store}>{children}</Provider>
  </ThemeWrapper>
)

export default TestProvider
