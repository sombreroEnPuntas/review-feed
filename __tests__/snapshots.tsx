import { render } from '@testing-library/react'
import React from 'react'

// pages
import Index from '../pages/index'

// Utils
import TestProvider from '../src/utils/TestProvider'

describe.each`
  Page
  ${Index}
`('$Page.displayName', ({ Page }) => {
  test('snapshot', () => {
    const { container } = render(
      <TestProvider>
        <Page />
      </TestProvider>
    )

    expect(container).toMatchSnapshot()
  })
})
