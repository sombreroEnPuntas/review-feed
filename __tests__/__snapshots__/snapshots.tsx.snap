// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Function Home] snapshot 1`] = `
.c12 {
  background-color: white;
  box-shadow: 4px 0 #4d5256, 0 -4px #4d5256, -4px 0 #4d5256, 0 4px #4d5256;
  display: block;
  margin: 0.5em;
  max-width: 450px;
  overflow-x: auto;
  padding: 0.5em;
  white-space: pre-wrap;
}

.c13 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #212529;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-family: 'Press Start 2P',monospace;
  font-size: 0.65em;
  font-weight: bold;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  line-height: calc(2 * 1.5em);
}

.c18 {
  background-color: white;
  box-shadow: 4px 0 #4d5256, 0 -4px #4d5256, -4px 0 #4d5256, 0 4px #4d5256;
  color: #212529;
  cursor: pointer;
  display: block;
  font-family: 'Press Start 2P',monospace;
  font-size: 0.65em;
  font-weight: bold;
  line-height: calc(2 * 1.5em);
  margin: 0.5em;
  padding: 0.5em;
}

.c15 {
  background-color: white;
  border: none;
  box-shadow: 0 4px #4d5256, 0 -4px #4d5256, 4px 0 #4d5256, -4px 0 #4d5256;
  font-family: 'Press Start 2P',monospace;
  line-height: calc(2 * 1.5em);
  margin: 4px;
  outline-color: #212529;
  padding: 0.5rem 1rem;
}

.c11 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #4d525666;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  left: 0;
  overflow: auto;
  position: fixed;
  top: 0;
  width: 100%;
  z-index: 1;
}

.c17 {
  font-size: 1em;
  line-height: 1.5em;
  margin: 1em 0;
  min-height: 3em;
}

.c16 {
  margin: 1em 0 0 0;
}

.c14 {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
}

.c1 {
  background-color: #3C3B41;
  padding: 1em;
}

.c2 {
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c3 {
  color: #ADACB1;
  margin: 0 0 0.5em 0;
  max-width: -webkit-max-content;
  max-width: -moz-max-content;
  max-width: max-content;
  padding: 0;
}

.c4 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #D8D7D9;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 20px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 50px;
}

.c4:before {
  background-color: #92cc41;
  content: '';
  display: inline-block;
  height: 5px;
  width: 20px;
}

.c5 {
  background-color: #D8D7D9;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c6 {
  background-color: #c9d68f;
  box-shadow: 4px 0 #3C3B41,0 -4px #3C3B41,-4px 0 #3C3B41,0 4px #3C3B41;
  color: #76c442;
  height: 4em;
  margin: 1em;
  overflow: auto;
  padding: 0.25em;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin-top: 0.5em;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c9 {
  background-color: white;
  box-shadow: 4px 0 #4d5256, 0 -4px #4d5256, -4px 0 #4d5256, 0 4px #4d5256;
  color: #212529;
  cursor: pointer;
  display: block;
  font-family: 'Press Start 2P',monospace;
  font-size: 0.65em;
  font-weight: bold;
  line-height: calc(2 * 1.5em);
  margin: 0.5em;
  padding: 0.5em;
  background-color: #ADACB1;
  box-shadow: none;
}

.c10 {
  background-color: white;
  box-shadow: 4px 0 #4d5256, 0 -4px #4d5256, -4px 0 #4d5256, 0 4px #4d5256;
  color: #212529;
  cursor: pointer;
  display: block;
  font-family: 'Press Start 2P',monospace;
  font-size: 0.65em;
  font-weight: bold;
  line-height: calc(2 * 1.5em);
  margin: 0.5em;
  padding: 0.5em;
  background-color: #E4303E;
  box-shadow: none;
}

.c0 {
  margin: auto;
  max-width: 450px;
  position: -webkit-sticky;
  position: sticky;
  top: 0;
  z-index: 1;
}

.c19 {
  margin: auto;
  max-width: 450px;
  padding: 0 0.5em;
  position: relative;
  z-index: 0;
}

<div>
  <main>
    <header
      class="c0"
    >
      <div
        class="c1"
      >
        <div
          class="c2"
        >
          <h3
            class="c3"
          >
            review-feed
          </h3>
          <div
            class="c4"
          />
        </div>
        <div
          class="c5"
        >
          <div
            class="c6"
          >
            Listing all themes.
          </div>
        </div>
        <div
          class="c7"
        >
          <div
            class="c8"
          >
            <button
              class="c9"
            >
              Filter
            </button>
            <button
              class="c9"
            >
              Reset
            </button>
          </div>
          <button
            class="c10"
          >
            Logout
          </button>
          <div
            class="c11"
          >
            <pre
              class="c12"
              type="black"
            >
              <code
                class="c13"
                type="black"
              >
                <form
                  class="c14"
                >
                  <input
                    aria-label="username"
                    autocomplete="username"
                    class="c15"
                    name="username"
                    placeholder="username"
                    required=""
                    type="text"
                    value=""
                  />
                  <div
                    class="c16"
                  />
                  <input
                    aria-label="password"
                    autocomplete="password"
                    class="c15"
                    name="password"
                    placeholder="password"
                    required=""
                    type="password"
                    value=""
                  />
                  <div
                    class="c17"
                  >
                    


                  </div>
                  <button
                    class="c18"
                  >
                    Login
                  </button>
                </form>
              </code>
            </pre>
          </div>
        </div>
      </div>
    </header>
    <section
      class="c19"
    />
  </main>
</div>
`;
